error_tracker/django/migrations/0002_auto_20201018_1311.py

```diff
@@ -6,7 +6,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('error_tracker', '0001_initial'),
```

Comment:
this is not correct, can you please verify if this is working? For me it's failing.

Reply:
i open an issue on it : #16.
Not working for me too...

Reply:
How are you adding error_tracker in the INSTALLED_APPS list?

Reply:
yes...
But with **django** as name in django_migrations on a previous setup... :-).
I cleanup all, and rerun... no more result...

Note : i need to place **error_tracker.django** to INSTALLED_APPS to get error_tracker working... and not the documentation specs...

Reply:
Can you please use `error_tracker.DjangoErrorTracker` in the installed app and verify?

Reply:
Also please share any exceptions if you use `error_tracker.DjangoErrorTracker` in the installed app.

Reply:
```
raise ImportError(_ERR_MSG.format(name), name=name)
ImportError: No module named 'DjangoErrorTracker'
```

Reply:
I think in new way of declaring modules in django, this line should be présent into __init__.py of the module
```
default_app_config = 'error_tracker.apps.DjangoErrorTracker'
```
I wrote all my modules like this.

Reply:
Can you please share python and Django version, this error is very strange.

Reply:
local env (mac OS) :
python : 3.8.5
Django : 3.1.1

Reply:
Unfortunately this is working fine for me on My Mac and same Django/Python version. 
Can you please take master pull and install it via `python setup.py install` and see if you're still getting similar error?

Reply:
i will git it a try